jx_sqlite/insert_table.py

```diff
@@ -238,6 +238,7 @@ def _flatten(data, uid, parent_id, order, full_path, nested_path, row=None):
             if not isinstance(data, Mapping):
                 data = {".": data}
             for k, v in data.items():
+                insertion = doc_collection[nested_path[0]]                
```

Comment:
I believe this is needed as when we iterate through data , after visiting the nested object(first sibling) , the second sibling nested object is inserted into previous sibling's active columns rather than to the fact `"."`. This can be confirmed by looking into the final return value of the flatten_many function as the active column record of `"a._c"`  has `a._b.$object` which should be in active column of  `"."` .
Yes the test even passes without this, But I think this is correct.